homeassistant/components/sma/sensor.py

```diff
@@ -168,6 +170,9 @@ def __init__(
             self._attr_state_class = STATE_CLASS_MEASUREMENT
             self._attr_device_class = DEVICE_CLASS_ENERGY
             self._attr_last_reset = dt_util.utc_from_timestamp(0)
+        if self.unit_of_measurement == POWER_WATT:
+            self._attr_state_class = STATE_CLASS_MEASUREMENT
+            self._attr_device_class = DEVICE_CLASS_POWER
```

Comment:
Are all the sensors current values? Or does it provides power sensors with daily average/min/max as well?

Reply:
I had briefly checked https://www.home-assistant.io/integrations/sma/ and it seems everything reporting POWER_WATT is a momentary measurement.

The code owners might be best suited to confirm.

Reply:
Correct, the SMA does not provide any average/min/max values as far as I know. Everything is instant/current values.